MetricsUploader/proto/orbit_log_event.proto

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) 2020 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+syntax = "proto3";
+
+package orbit_metrics_uploader;
+
+// This proto is used for communication between Orbit and metrics uploader
+// client. Changing this proto requires changing the metrics uploader client
+// library accordingly.
+message OrbitLogEvent {
+  enum LogEventType {
+    UNKNOWN_EVENT_TYPE = 0;
+    ORBIT_INITIALIZED = 1;
+    ORBIT_CAPTURE_DURATION = 2;
```

Comment:
Here I've used the similar to others teams approach (to have a separate log type for every event). Having a comment describing an event could be another option, but what's advantage of it?

Reply:
It would potentially give more flexibility as you don't have to change the LogEventType definition if you introduce a new event somewhere. But if this is common practice, it seems fine to me.